ttnn/cpp/ttnn/operations/core/core.hpp

```diff
@@ -69,7 +69,7 @@ ttnn::Tensor allocate_tensor_on_device(
 
 void copy_host_to_device_tensor(ttnn::Tensor host_tensor, ttnn::Tensor device_tensor, uint8_t cq_id = 0);
 
-ttnn::Tensor from_device(const ttnn::Tensor& tensor, bool blocking = true);
+ttnn::Tensor from_device(const ttnn::Tensor& tensor, bool blocking = true, uint8_t cq_id = 0);
```

Comment:
we have DefaultQueueId somewhere instead of 0.

Reply:
All instances of `uint8_t cq_id = 0` related to this PR have been updated